data/tab-category.json

```diff
@@ -62,6 +62,20 @@
     "link": null
 },
 {
+    "tab": "starters",
+    "category": "rust-cookbook",
+    "labels": ["easy"],
+    "milestone": null,
+    "link": null
+},
+{
+    "tab": "impl",
+    "category": "rust-cookbook",
+    "labels": [],
+    "milestone": null,
```

Comment:
Do you have some milestone or label for filtering the issues? I'd prefer to avoid using all issues from a repository - this kind of thing is most effective when there is a small, curated set of issues. People can browse the repo if they want to find more.

Reply:
@nrc Ive updated the "impl" tab with labels "example" and "tracking issue" (i hope that these are additive) to filter out the noise.